src/io/flutter/view/FlutterView.java

```diff
@@ -180,6 +187,17 @@ private void addInspector(FlutterApp app, InspectorService inspectorService, Too
                       true);
     addInspectorPanel("Render Tree", tabs, state, InspectorService.FlutterTreeType.renderObject, app, inspectorService, toolWindow,
                       toolbarGroup, false);
+
+    final boolean isVertical = !toolWindow.getAnchor().isHorizontal();
```

Comment:
Smart!  Curious what happens if you change orientations when the view is rendered.  (I'm guessing nothing bad and that you just get the appropriate settings on next run.)

Reply:
Incidentally, another thing I was considering was making the displays themselves be (re)size-aware.  For example, they could widen given more space and/or show more details (like summaries, etc.) . Anyway, more to consider!

Reply:
Yeah, it updates on the next app run. We should definitely plan to make this more dynamic in the future.